.github/workflows/ci.yaml

```diff
@@ -73,6 +73,10 @@ jobs:
           python-version: "3.x"
           architecture: x64
 
+      - run: |
```

Comment:
Instead of a separate job, let's add a `pre-commit` hook to [`.pre-commit-config.yaml`](https://github.com/bachya/ecowitt2mqtt/blob/dev/.pre-commit-config.yaml). A quick Google yielded [this library](https://github.com/gruntwork-io/pre-commit), which provides a `helmlint` hook (might be worth glancing at their [caveats](https://github.com/gruntwork-io/pre-commit#helm-lint-caveats)).

Doing it this way brings two advantages:

1. It will automatically be run via the `pre-commit/action@v3.0.0` CI action.
2. It will be run locally when a future dev modifies the Helm chart (and encourage them to fix any issues before the PR is submitted).